diff --git a/src/Banho.java b/src/Banho.java
new file mode 100644
index 0000000..7ba88f2
--- /dev/null
+++ b/src/Banho.java
@@ -0,0 +1,2 @@
+public class Banho {
+}
diff --git a/src/BanhoTosa.java b/src/BanhoTosa.java
new file mode 100644
index 0000000..817b498
--- /dev/null
+++ b/src/BanhoTosa.java
@@ -0,0 +1,2 @@
+public class BanhoTosa {
+}
diff --git a/src/Exame.java b/src/Exame.java
new file mode 100644
index 0000000..b1e96c3
--- /dev/null
+++ b/src/Exame.java
@@ -0,0 +1,2 @@
+public class Exame {
+}
diff --git a/src/Medicamento.java b/src/Medicamento.java
new file mode 100644
index 0000000..64d32cc
--- /dev/null
+++ b/src/Medicamento.java
@@ -0,0 +1,2 @@
+package PACKAGE_NAME;public class Medicamento {
+}
diff --git a/src/Tosa.java b/src/Tosa.java
new file mode 100644
index 0000000..fb4efa4
--- /dev/null
+++ b/src/Tosa.java
@@ -0,0 +1,2 @@
+package PACKAGE_NAME;public class Tosa {
+}
diff --git a/src/Tosador.java b/src/Tosador.java
new file mode 100644
index 0000000..d090b19
--- /dev/null
+++ b/src/Tosador.java
@@ -0,0 +1,2 @@
+public class Tosador {
+}
diff --git a/src/Vacina.java b/src/Vacina.java
new file mode 100644
index 0000000..62889a4
--- /dev/null
+++ b/src/Vacina.java
@@ -0,0 +1,2 @@
+public class Vacina {
+}
